index.d.ts

```diff
@@ -2197,20 +2197,21 @@ export type FilesystemMountpoint = {
  * Filesystem Adapter Methods
```

Comment:
Gah. I can't see if these are tabs or note here. But wasn't this fixed in another commit ?